src/App.vue

```diff
@@ -77,55 +121,79 @@ export default {
   // beforeDestroy() {
   //   document.removeEventListener("keydown", this._keyListener);
   // },
-
+  watch: {
+    rewardType: function (newReward) {
+      this.giphyImgSrc = null;
+      this.giphyImgSrcA = null;
+      this.giphyImgSrcB = null;
+      this.giphyToggle = false;
+      this.loadGiphy(newReward);
+      console.log('changed preloaded giphy to' + newReward);
+    }
+  },
   methods: {
-
     addTask(e) {
       let taskObject = { id: this.taskId++, name: e, status: false };
```

Comment:
it would be more robust to generate a uuid for the task id:
```
> const uuid = require('uuid')
undefined
> uuid.v4()
'5c6aee94-b95b-4a9a-b91f-4daf903c1b0c'
```
then you also wouldn't need to track `this.taskId` to know which task ids have been taken